dev/deps/spark-deps-hadoop-2.2

```diff
@@ -116,7 +116,7 @@ libfb303-0.9.2.jar
 libthrift-0.9.2.jar
 log4j-1.2.17.jar
 lz4-1.3.0.jar
-mesos-0.21.1-shaded-protobuf.jar
+mesos-0.28.2-shaded-protobuf.jar
```

Comment:
It's much less of an issue than in other projects, because the library itself talking to the Mesos master.  Libmesos is.  The risk here is incompatibility with existing libmesos.  That layer is actually extremely stable.  I haven't seen any incompatibilities in any of our frameworks.  But once Mesos releases 1.0 (it's in RC now), this should be less of an issue.